src/content/en/fundamentals/performance/rail.md

```diff
@@ -1,171 +1,350 @@
+      <td data-th="User Reaction">Within this window, things feel part of a natural and
+      continuous progression of tasks. For most users on the web, loading pages or changing
+      views represents a task.</td>
     </tr>
     <tr>
-      <td data-th="Delay">1000+ms</td>
-      <td data-th="User Reaction">Beyond 1 second, the user loses focus on the task they are performing.</td>
+      <td data-th="Delay">1000ms or more</td>
+      <td data-th="User Reaction">Beyond 1000 milliseconds (1 second), users lose focus on
+      the task they are performing.</td>
     </tr>
     <tr>
-      <td data-th="Delay">10,000+ms</td>
-      <td data-th="User Reaction">The user is frustrated and is likely to abandon the task; they may or may not come back later.</td>
+      <td data-th="Delay">10000ms or more</td>
+      <td data-th="User Reaction">Beyond 10000 milliseconds (10 seconds), users are frustrated
+      and are likely to abandon tasks. They may or may not come back later.</td>
     </tr>
   </tbody>
 </table>
 
-## Response: respond in under 100ms {: #response }
+Users perceive performance delays differently, depending on network conditions and hardware.
+For example, loading an experience in 1000ms is plausible on a powerful desktop machine over a
+fast Wi-Fi connection, so users have grown accustomed to a 1000ms loading experience. But for
+mobile devices over slow 3G connections, loading in 5000ms is a more realistic goal, so mobile
+users are generally more patient.
 
-You have 100ms to respond to user input before they notice a lag.
-This applies to most inputs, such as clicking buttons, toggling form
-controls, or starting animations. This does not apply to touch drags or
-scrolls.
+## Response: respond in under 50ms {: #response }
 
-If you don't respond, the connection between action and reaction is broken. Users will notice.
+**Goal**: Complete a transition initiated by user input within 100ms. Users spend the majority
+of their time waiting for sites to respond to their input, not waiting for the sites to load.
 
-While it may seem obvious to respond to user's actions immediately,
-that's not always the right call.
-Use this 100ms window to do other expensive work, but be careful not to block the user.
-If possible, do work in the background.
+**Guidelines**:
 
-For actions that take longer than 500ms to complete, always provide feedback.
+* Respond to user input within 50ms, or else the connection between action and
+  reaction is broken. This applies to most inputs, such as clicking buttons, toggling form
+  controls, or starting animations. This does not apply to touch drags or scrolls.
+* Though it may sound counterintuitive, it's not always the right call to respond to user
+  input immediately. You can use this 100ms window to do other expensive work. But be
+  careful not to block the user. If possible, do work in the background.
+* For actions that take longer than 500ms to complete, always provide feedback.
```

Comment:
Is this supposed to be 50ms?

Reply:
You're absolutely right - this should be 50ms.